include/stdexec/execution.hpp

```diff
@@ -3297,9 +3325,8 @@ namespace stdexec {
     struct split_t {
       template <sender _Sender, class _Env = __empty_env>
           requires
-            (copy_constructible<remove_cvref_t<_Sender>> &&
-             sender_to<_Sender&, __receiver_t<_Sender, _Env>>) ||
-            __is_split_customized<_Sender, _Env>
+             (sender_in<_Sender, _Env> && constructible_from<decay_t<attrs_of_t<_Sender>>, attrs_of_t<_Sender>>) ||
```

Comment:
> `constructible_from<decay_t<attrs_of_t<_Sender>>, attrs_of_t<_Sender>>`

We should have a `__decay_copable` concept that simplifies this. It would look like:

```c++
template <class _Ty>
  concept __decay_copyable =
    constructible_from<decay_t<_Ty>, _Ty>;
```

... and it would live along side (and be subsumed by) `__nothrow_decay_copyable` in concepts.hpp.